Assets/Scripts/Interactables/Interactable.cs

```diff
@@ -82,13 +93,13 @@ public void FaceInteractable(GameObject player)
     ///////// WILL CHANGE ANIMATION LATER (FADE IN/OUT FOR NOW)
     private void ShowInteractUI()
     {
-        interactableAnim.SetTrigger("FadeIn");
+        interactableAnim.SetTrigger("PopIn");
```

Comment:
Ah, just remembered, can we add this (and the string below for "PopDown") in Constants.cs, and then reference it there? To keep things clean?